textworld/envs/glulx/git_glulx.py

```diff
@@ -0,0 +1,150 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license.
+
+
+# -*- coding: utf-8 -*-
+import os
+import re
+import sys
+import textwrap
+import subprocess
+from pkg_resources import Requirement, resource_filename
+
+from typing import Mapping, Union, Tuple, List
+
+import numpy as np
+
+from glk import ffi, lib
+from io import StringIO
+
+import textworld
+from textworld.core import GameState
+from textworld.core import GameNotRunningError
+
+GLULX_PATH = resource_filename(Requirement.parse('textworld'), 'textworld/thirdparty/glulx/Git-Glulx')
+
+
+
+def _strip_input_prompt_symbol(text: str) -> str:
+    if text.endswith("\n>"):
+        return text[:-2]
+
+    return text
+
+
+class GitGlulxEnv(textworld.Environment):
+    """ Environment to support playing Glulx games.
+
+    This environment supports playing text-based games that were compiled for
+    the `Glulx virtual machine <https://www.eblong.com/zarf/glulx>`_. The main
+    advantage of using Glulx over Z-Machine is it uses 32-bit data and
+    addresses, so it can handle game files up to four gigabytes long. This
+    comes handy when we want to generate large world with a lot of objects
+    in it.
+
+    We use a customized version of `git-glulx <https://github.com/DavidKinder/Git>`_
+    as the glulx interpreter. That way we don't rely on stdin/stdout to
+    communicate with the interpreter but instead use UNIX message queues.
```

Comment:
This is a lie, and anyway I don't think we need to document it.  (I know this was just cut and pasted, but let's clean it up anyway)

Reply:
Haha, good point. That should have been updated. What would be an accurate description for what we are doing?

Reply:
UNIX sockets?